projects/storefrontlib/src/cms-components/product/product-variant-selector/style-icons/style-icons.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+import {
+  OccConfig,
+  VariantOption,
+  VariantOptionQualifier,
+  VariantQualifier,
+} from '@spartacus/core';
+
+@Component({
+  selector: 'cx-style-icons',
+  templateUrl: './style-icons.component.html',
+  styleUrls: ['./style-icons.component.scss'],
```

Comment:
It is possible to make it with ` changeDetection: ChangeDetectionStrategy.OnPush,`?